test/helper.lua

```diff
@@ -237,4 +237,21 @@ function helpers.is_expired_debug(_, tuple)
     return true
 end
 
+function helpers.tarantool_version()
+    local major_minor_patch = _G._TARANTOOL:split('-', 1)[1]
+    local major_minor_patch_parts = major_minor_patch:split('.', 2)
+
+    local major = tonumber(major_minor_patch_parts[1])
+    local minor = tonumber(major_minor_patch_parts[2])
+    local patch = tonumber(major_minor_patch_parts[3])
+
+    return major, minor, patch
+end
+
+function helpers.vinyl_is_broken()
+    -- Blocked by https://github.com/tarantool/tarantool/issues/6609
+    local major, minor = helpers.tarantool_version()
+    return major == 2 and (minor >= 1 or minor <= 7)
```

Comment:
Bug should be fixed for some 1.10 versions. And I think it should be reflected in condition.

Reply:
Fixed.

```diff
--- a/test/helper.lua
+++ b/test/helper.lua
@@ -251,10 +251,9 @@ end
 function helpers.vinyl_is_broken()
     -- Blocked by https://github.com/tarantool/tarantool/issues/6609
     local major, minor, patch = helpers.tarantool_version()
-    -- 1.10.11-5+
-    local v1_10 = major == 1 and minor >= 10 and patch >= 11
-    local v2 = major == 2 and (minor >= 1 or minor <= 7)
-    return v1_10 or v2
+    local broken_v1_10 = major == 1 and minor >= 10 and patch >= 11
+    local broken_v2 = major == 2 and (minor >= 1 or minor <= 7)
+    return broken_v1_10 or broken_v2
 end
 
 return helpers
```